shuttle/tests/basic/replay.rs

```diff
@@ -29,22 +29,26 @@ fn concurrent_increment_buggy() {
 }
 
 #[test]
+#[ignore = "replay mechanism is broken because the schedule is not emitted in the panic output. reintroduce once replay mechanism is fixed."]
```

Comment:
What's the plan here? Could we define a `replay_with_expected_panic` for these tests at least?